contracts/Freeport.sol

```diff
@@ -23,9 +23,19 @@ contract Freeport is /*
     TransferFees,
     */ SimpleExchange {
 
+    /** Name of the collection for open sea.
```

Comment:
This looks like a copy/paste from another PR. Be careful because this will lead to errors introduced in merges. It’s better to build on top of other commits (branch of branch) rather than copy/paste.